### PR TITLE
[FLINK-26634][docs-zh] Update Chinese version of Elasticsearch connector docs

### PR DESCRIPTION
**Pull request by chenzihao5**
## What is the purpose of the change

Update Chinese version of Elasticsearch connector docs as it is described in https://issues.apache.org/jira/browse/FLINK-26634. 

The page is https://nightlies.apache.org/flink/flink-docs-master/zh/docs/connectors/datastream/elasticsearch/

The markdown file is located in `docs/content.zh/docs/connectors/datastream/elasticsearch.md`

## Brief change log

- Translate `docs/content.zh/docs/connectors/datastream/elasticsearch.md`

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no

### CODE REVIEW

**Review by hackergin — COMMENTED** [review]
@chenzihao5  Thanks for your contribution, I left some comments.

File: docs/content.zh/docs/connectors/datastream/elasticsearch.md [hunk 1]
```diff
@@ -210,72 +210,35 @@ def createIndexRequest(element: (String)): IndexRequest = {
...
-  .setHosts(new HttpHost("127.0.0.1", 9200, "http"))
-  .setEmitter((element: String, context: SinkWriter.Context, indexer: RequestIndexer) =>
-  indexer.add(createIndexRequest(element)))
 ```
 {{< /tab >}}
 {{< /tabs >}}
 
+<p style="border-radius: 5px; padding: 5px" class="bg-info">
+<b>重要提示</b>: 默认情况下不启用 checkpoint, 但默认传输保证 AT_LEAST_ONCE 语义。
+这会导致接收器缓冲请求直到完成，或 BulkProcessor 自动刷新。
```
[the remark this answers is not in the document]
Comment (chenzihao5) [reply docs/content.zh/docs/connectors/datastream/elasticsearch.md:238]: Thanks for your revisions. Please remove the space after the comma.

**Comment by chenzihao5** [comment]
Hi, @gaoyunhaii . Can you help to review this? Thanks a lot.

**Comment by gaoyunhaii** [comment]
Hi @chenzihao5 very thanks for the PR! The ElasticSearch should have been changed a bit due to some compatibility issue in the past days and the document also has been modified a bit, could you also rebase the PR to the latest master?

**Comment by chenzihao5** [comment]
Hi, @gaoyunhaii . Thanks for your review. I have rebased this PR to the latest master.

**Review by gaoyunhaii — APPROVED** [review]
LGTM, thanks @chenzihao5 for the PR and @hackergin for the review!

**Pull request closed without merging**